Lib/test/test_msvcrt.py

```diff
@@ -115,6 +115,28 @@ def test_heap_min(self):
         except OSError:
             pass
 
+    def test_GetErrorMode(self):
+        msvcrt.GetErrorMode()
+
+    def test_SetErrorMode(self):
+        old = msvcrt.SetErrorMode(0)
+        msvcrt.SetErrorMode(old)
```

Comment:
All of these tests do not assert anything. Can we improve that?

Reply:
Thanks for the reivew. These functions are just wrappers of the MSVCRT's APIs, and they just return int values, which may be changed by Microsoft or by other test cases.

So, if we want to keep the test cases stable, what we do here may be just to assert that they are ints. But I think this doesn't provide too much help for testing. What do you think?